.env

```
@@ -3,3 +3,7 @@ EXPO_PUBLIC_VIBECODE_ANTHROPIC_API_KEY=sk-ant-api03-gu2gohc4sha1Thohpeep7ro9vie1
 EXPO_PUBLIC_VIBECODE_GROK_API_KEY=xai-ahDi8ofei1Em2chaichoac2Beehi8thu-n0tr3al
 EXPO_PUBLIC_VIBECODE_GOOGLE_API_KEY=UeHoh2oot2IWe6ooW4Oofahd6waebeiw-n0tr3al
 EXPO_PUBLIC_VIBECODE_ELEVENLABS_API_KEY=elevenlabs-api-key-oa9Shahx4Zi4oof2bei5kee9nee7eeng-n0tr3al
+
+# Supabase Configuration
+EXPO_PUBLIC_VIBECODE_SUPABASE_URL=https://xhtqobjcbjgzxkgfyvdj.supabase.co
+EXPO_PUBLIC_VIBECODE_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InhodHFvYmpjYmpnenhrZ2Z5dmRqIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTQ1NDg3MjAsImV4cCI6MjA3MDEyNDcyMH0.pRMiejad4ILuHM5N7z9oBMcbCekjSl-1cM41lP1o9-g
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_VIBECODE_SUPABASE_ANON_KEY key should go before the EXPO_PUBLIC_VIBECODE_SUPABASE_URL key

(UnorderedKey)